pkg/controller/resourcequota/resource_quota_monitor.go

```diff
@@ -132,7 +134,7 @@ type monitors map[schema.GroupVersionResource]*monitor
 
 func (qm *QuotaMonitor) controllerFor(resource schema.GroupVersionResource) (cache.Controller, error) {
 	// TODO: pass this down
-	clock := clock.RealClock{}
+	realClock := clock.RealClock{}
```

Comment:
Just inline this. It's typedef'd to an `struct{}` so it doesn't allocate. There's no reason to save it as a local.